utils/results/conversion/sarifparser/sarifparser.go

```diff
@@ -52,15 +50,42 @@ type CmdResultsSarifConverter struct {
 	patchBinaryPaths bool
 	// Current stream parse cache information
 	current       *sarif.Report
-	scaCurrentRun *sarif.Run
-	currentTarget results.ScanTarget
-	parsedScaKeys *datastructures.Set[string]
+	currentTarget *currentTargetState
```

Comment:
i'd change the name here to currentTargetState so we will not have a mixup with currentTarget inside currentTargetState